CONTRIBUTING.md

```diff
@@ -37,13 +37,17 @@ So **you are "the community"**, go and help others!
 
 https://community.privacyidea.org
 
+## Long Term Supported Versions
+We currently do **not** have a long term supported versions. If a new minor version is released, 
+the previous minor versions will usually not receive any more updates. For pressing matters, there might be exceptions.
```

Comment:
the whole paragraph?

Reply:
No, just "For pressing matters, there might be exceptions."...
Since everyone thinks that his aspect is pressing. ...and we never did this, so I would just remove this very sentence.

Reply:
ok